modules/apps/commerce/commerce-frontend-js/test/components/AccountSelector/AccountSelector.js

```diff
@@ -103,9 +103,10 @@ describe('AccountSelector', () => {
 			const accountsList = renderedComponent.baseElement.querySelectorAll(
 				'.accounts-list li'
 			);
+
 			const accountsListItem = accountsList[0];
 
-			expect(accountsList.length).toBe(10);
+			expect(accountsList.length).toBe(11);
```

Comment:
This is a bit of a smell here. Why would this test need updating when you never made changes to the AccountSelector component?

Reply:
`AccountSelector` is the parent component, and I've added a new `ClayDropdown.Section` (the one with the button to open the modal) which I believe is an `li` which in turn increases the amount of `li`s in the component, making me have to update the test.

Reply:
Ah, I see. Looks like the original test then is a bit weird then, because to count based on the number of `li` seems flakey. In this case I think you're fine then, I wouldn't be over-eager and re-write the test. maybe one day...